debug/debug.go

```diff
@@ -55,9 +57,15 @@ func StartStackTraceHandler() {
 				"\n====== STACKTRACE ======\n%v\n%s\n====== /STACKTRACE ======\n",
 				time.Now(),
 				stackDump,
-			), journal.PriDebug)
+			), journal.PriDebug, 2)
 		}
 
 		os.Exit(1)
 	}()
 }
+
+func DeferFuncForRunLogDriver() {
+	if LoggerErr != nil {
+		SendEventsToJournal(daemonName, LoggerErr.Error(), journal.PriErr, 1)
```

Comment:
When we call `SendEventsToJournal()`, we choose the hard coded value of 1 or 2 seconds as the delay. Would be helpful if we set a `const` variable for the delay time, so it's easier to modify in the future, in case we want to sleep for longer time for some cases. 

Also I think it's worth documenting how long we sleep to wait for the journald logs in the README.

Reply:
The reason why I did not set it as a constant because on debug mode, we will log every log message scanned from pipes, and it would be a disaster if we wait 1 seconds after sending every log message to journald. I can update the readme with this info.